### ISSUE TITLE
H5BP v5.0 Fork Available

### ISSUE THREAD

**Issue by tohuw** [sporkd/compass-h5bp#26]
I've forked this and updated it to support everything from version 5.0 of H5BP.

Because it has been some time since this project has been touched, I renamed my fork and proceeded with publishing it as a separate gem, [compass-html5bp](https://rubygems.org/gems/compass-html5bp).

However, Peter, if you're still interested in maintaining this project, I'm more than happy to refactor [my fork](https://github.com/tohuw/compass-html5bp) into something suitable for a pull request, then deprecate compass-html5bp, in true Highlander-esque fashion.

Please do let me know if I've done anything improper in this process - a Rubyist I am not. I just love Compass, and thought the work you did here was fantastic and deserved an update.

**Comment by tohuw** [comment]
Per your request on Twitter, I've created a pull request. Upon merge, and after any additional mess is hashed out, I'd like to deprecate my gem - I don't think anyone is really using it at this point anyway.

I've bumped the version to 1.0.0, as SemVer would dictate for this, as the mixin names have changed and as a whole this will not be a backwards-compatible change. However, thanks to your already existing tags, and given a tag for this version, users ought to be able to navigate the migration path.

I tried to document the changes thoroughly. Let me know what else I can provide.
